### DISCUSSION TITLE
How does this work?

### DISCUSSION THREAD

**Discussion by cparker4486** [nothingislost/obsidian-hover-editor#228] [Q&A] [answered]
I've installed the plugin, enabled it, and set some options that seem appropriate to me but nothing happens. I closed and reopened Obsidian but still nothing. I'm a new Obsidian user so I'm only using `[[Note]]` and `![[Note]]` thus far. I thought it would work automatically for the embedded note. What should I be doing instead?

Thanks!

**Comment by pjeby** [comment]
I don't know what you mean by "work" here.  There is a Hover Editor setting to enable it to trigger on embedded notes, though.  Maybe you need to turn that on?

**Comment by cparker4486** [comment]
"Work" meaning no popup or hover editor appears. Nothing happens that is different from not using the plugin and that's why I'm confused. It seems like it should be working already.

The Hover Editor plugin itself is enabled, "Default Mode" is "Editing view", and "Trigger hover preview on embeds" is enabled. But nothing different happens in Obsidian when I embed a note with `![[Note]]`. I just see the embedded note like I always have.

Am I still missing something?

**Comment by pjeby** [comment, marked as answer]
No popup appears when you do what, exactly?

Here's what should happen:

- If you hover a link while holding Ctrl, you get a popup
- If you hover an embed (and the option is enabled) while holding Ctrl, you get a popup

If you aren't hovering the mouse over a link or embed, or you aren't holding Ctrl, nothing is going to happen.  If you don't want to have to hold Ctrl, then you need to go to the Page Preview settings (under "Core plugins") and turn off the toggle for any place where you want to  get a popup without holding Ctrl.

**Reply by cparker4486** [reply]
Didn't know I had to hold "Ctrl". Now it's working. Thanks.
